cvss_util/utils/test.py

```diff
@@ -1,9 +1,73 @@
 #!/usr/bin/python
 import unittest
 
+from . import create_cvss_format
 from . import parse_cvss_format
 
 
+class TestCVSSFormatter(unittest.TestCase):
+
+    def test_format_critical_cvss_v2(self):
+        """ tests creating a jira table comment for a critical cvss v2 """
+        vector_string = 'AV:N/AC:L/Au:N/C:P/I:P/A:C'
+        expected = """
+Proposed CVSS score: 9.0 => *Critical* severity
+
+*Exploitability Metrics*
+
+|| AccessVector | Network |
+|| AccessComplexity | Low |
+|| Authentication | None |
+
+
+*Impact Metrics*
+
+|| ConfImpact | Partial |
+|| IntegImpact | Partial |
+|| AvailImpact | Complete |
+
+"""
+        self.assertEqual(
+            create_cvss_format.cvss_v2_vector_to_jira_table(
+                vector_string,
+                9.0,
+            ),
+            expected
```

Comment:
@ashleyblackmore what you could do is test parsing the generated output with the comment parsing code.